tests/react-test-app/package.json

```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "react": "15.3.2",
     "react-native": "^0.37.0",
+    "react-native-fs": "^1.1.0",
```

Comment:
this is needed for the `react-test-app` which run tests for Android `tests/react-test-app/android/settings.gradle`. We use it in `tests/react-test-app/index.android.js` to export the test result to the SD card on Android emulator.